core/server/api/posts.js

```diff
@@ -10,7 +10,7 @@ var Promise = require('bluebird'),
     allowedIncludes = [
         'created_by', 'updated_by', 'published_by', 'author', 'tags', 'fields'
     ],
-    unsafeAttrs = ['author_id'],
+    unsafeAttrs = ['author_id', 'status'],
```

Comment:
Just leaving this note here.

The whole concept of `unsafeAttrs` is a bit confusing. So the purpose is to make the model permission layer dynamic. You can pass attributes, which can change the outcome. 

But the whole implementation contradicts:

e.g. the `status` is a restricted field for contributors. You can't publish.
So if you don't pass `unsafeAttrs` (different use case in the code base), the contributor would be able to publish a post. That does not makes sense to me 🤔

Reply:
Yeah - the fact that unsafeAttrs is defined in the controller rather then at the model layer did seem a bit confusing to me too; however I'd say that changes to that would probably be outside the scope of this PR (especially as it can affect other places not related to the contributor role.

Reply:
>  I'd say that changes to that would probably be outside the scope of this PR 

Definitely 👍 Just something to keep in mind or even worth raising an issue.